### PR TITLE
Include root level json files in `reformat-files`

### PR DESCRIPTION
**Pull request by simison**
- Include root level json files in `reformat-files`
- ...and run `yarn reformat-files`

Follow up to https://github.com/Automattic/jetpack/issues/11507

Spaces in package.json seem to be there for historic reasons; NPM enforced them until NPM v5 and nowadays NPM doesn't care.

#### Changes proposed in this Pull Request:
- Include json file extension when running prettier on root level files

#### Testing instructions:
Running `yarn reformat-files` should fix spaces in JSON files in the root folder.

#### Proposed changelog entry for your changes:
N/A

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,211 +1,211 @@
...
+		"php:compatibility": "composer php:compatibility",
+		"php:lint": "composer php:lint",
+		"php:autofix": "composer php:autofix",
+		"test-adminpage": "yarn test-client && yarn test-gui",
+		"test-client": "NODE_ENV=test NODE_PATH=tests:_inc/client:node_modules/@automattic/dops-components/client tests/runner.js",
+		"add-textdomain": "wpi18n addtextdomain --textdomain=jetpack --glob-pattern='!(docker|node_modules|tests|tools|vendor){*.php,**/*.php}'",
+		"test-gui": "NODE_ENV=test NODE_PATH=tests:_inc/client:node_modules/@automattic/dops-components/client tests/runner.js gui",
+		"test-modules": "NODE_ENV=test NODE_PATH=tests:_inc/client:node_modules/@automattic/dops-components/client tests/runner.js modules",
+		"test-dangerci-and-adminpage": "yarn danger ci && yarn test-adminpage",
+		"reformat-files": "./node_modules/.bin/prettier --ignore-path .eslintignore --write \"{_inc,bin,extensions,modules,tools,tests}/**/*.{js,jsx,json}\" ./*.{js,json}"
```
Comment (simison) [inline package.json]: The actual non-whitespace change is at this line.

**Review by jeherve — DISMISSED** [review]
👍 	🚢

File: package.json (hunk 1 above)
Comment (sirreal) [inline package.json]: Let's do it all, that's how pre-commit works.
```suggestion
		"reformat-files": "./node_modules/.bin/prettier --ignore-path .eslintignore --write \"**/*.{js,jsx,json}\""
```

**Comment by simison** [comment]
@jeherve this now needs another ✅ :-)

**Review by kraftbj — APPROVED** [review]

**Pull request merged**